apps/melding-form/.env

```diff
@@ -1 +1,2 @@
 NEXT_PUBLIC_BACKEND_BASE_URL=http://localhost:8000
+NEXT_BACKEND_BASE_URL=http://localhost:8000
```

Comment:
Will only be used in local development. 
For docker builds and cloud deployments we will overwrite these env vars.

Reply:
Alternative is to add a fallbak to localhost:8000 inside the runtime-config file for `NEXT_BACKEND_BASE_URL`

Reply:
- .env variabele toevoegen in backoffice 
- baseUrl verwijderen uit apiClientProxy.ts